src/components/MarkdownSyntaxModal.js

```diff
@@ -2,7 +2,7 @@ import { Fragment, useState } from 'react'
 import { Dialog, Transition } from '@headlessui/react'
 import { XMarkIcon } from '@heroicons/react/24/outline'
 
-export default function MarkdownSyntaxModal({open, setOpen}) {
+export default function SetupEPNSChannelModal({ open, setOpen }) {
```

Comment:
@tzumby  I think that is a wrong modal. Same for the Ttitle below
it's `MarkdownSyntaxModal`, not `SetupEPNSChannelModal`

Reply:
Oops :)